pipeline_dp/combiners.py

```diff
@@ -507,6 +559,17 @@ def create_compound_combiner(
         combiners.append(
             PrivacyIdCountCombiner(
                 CombinerParams(budget_privacy_id_count, aggregate_params)))
+    if pipeline_dp.Metrics.ARRAY_SUM in aggregate_params.metrics:
+        budget_vector_sum = budget_accountant.request_budget(
+            mechanism_type, weight=aggregate_params.budget_weight)
+        combiners.append(
+            VectorSumCombiner(
+                CombinerParams(budget_vector_sum, aggregate_params)))
+        if pipeline_dp.Metrics.SUM in aggregate_params.metrics or \
```

Comment:
I thought again, where to check, we should fail eariler, in this case in `[AggregateParams.__post_init()](https://github.com/OpenMined/PipelineDP/blob/3a7b87a5d943ea6b083c45934c8bb1110cbe5766/pipeline_dp/aggregate_params.py#L86)`. Could you please move it there?